standalone-metastore/DEV-README

```diff
@@ -0,0 +1,40 @@
+This file contains information for developers and testers.
+
+Testing metastore installation and upgrade against databases beyond Derby:
+There are integration tests for testing installation and upgrade of the
+metastore on MySQL (actually MariaDB is used), Oracle, Postgres, and SQLServer.
+These tests are not run by default because they take several minutes each and
+they require the developer to download the JDBC driver for Oracle.
+They are run in the integration-test phase.
+
+Each ITest runs two tests, one that installs the latest version of the
+database and one that installs the latest version minus one and then upgrades
+the database.
+
+To run one of the tests you will need to explicitly turn on integration testing,
+in the Oracle case specify the location of the JDBC driver, and optionally
+specify which test you want to run.  To run all of the tests do:
+
+mvn verify -Ditest.jdbc.jars=_oracle_jar_path -DskipITests=false -Dtest=nosuch
+
+To run just one test, do
+
+mvn verify -DskipITests=false -Dit.test=ITestMysql -Dtest=nosuch
+
+You can download the Oracle driver at 
+http://www.oracle.com/technetwork/database/features/jdbc/index-091264.html
+
+If you wish to use one of these containers to run your own tests against a
```

Comment:
are there any licensing implications of adding it to our source code?

Reply:
We can't distribute it. As long as it's used only in test scope I believe we are fine.  This is particularly true because we aren't directly linking to it, but just using it to execute the JDBC calls Hive makes.

Reply:
okay. Thanks for the clarification